.github/workflows/workflow.yml

```diff
@@ -0,0 +1,46 @@
+name: D2L License Checker CI
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
```

Comment:
Sorry not sure if I misunderstood - we should remove this so this workflow runs on all pushes / branches now right?

Reply:
If you don't give it an `on` value will it just run on anything?

Reply:
No I believe you need the event names:

```yaml
on:
  - pull_request
  - push
```

or 

```yaml
on:
  - pull_request:
  - push:
```